qiskit/visualization/timeline/types.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+r"""
+Special data types.
+
+- ScheduledGate:
+    t0: Time when the instruction is issued.
+    operand: Gate object associated with the instruction.
+    duration: Time duration of the instruction.
+    bits:
+
+- GateLink:
+    t0: Position where the link is placed.
+    operand: Gate object associated with the instruction.
+    bits: List of bit associated with the instruction.
+
+- Barrier:
+    t0: Position where the barrier is placed.
+    bits: List of bit associated with the instruction.
```

Comment:
Could this documentation be attached to the `NamedTuple`s https://stackoverflow.com/a/20388499?

Reply:
Thanks! I didn't notice this is supported by py3.5. Now it looks cleaner.